testplan/report/testing/schemas.py

```diff
@@ -195,11 +195,15 @@ class TestCaseReportSchema(ReportSchema):
 
     status_override = fields.String(allow_none=True)
 
+    display_name = fields.String(allow_none=True)
+
     entries = fields.List(EntriesField())
 
     status = fields.String(dump_only=True)
     runtime_status = fields.String(dump_only=True)
     counter = fields.Dict(dump_only=True)
+    passed = fields.Bool()
```

Comment:
We don't need these extra fields - we already send the status, this information is redundant.

Reply:
because in another PR you left a comment that don't check the pass or fail status for a testcase, it already have the attribute. There the report has already been dumped to json so no 'passed' or 'failed' key is kept. But I agree to remove the attributes here, because in a recent commit, it add an unstable status, for example, a skipped testcase is unstable, but if a suite has 2 cases: passed and unstable, then the suite is passed, so RFB cannot reply on 'passed' or 'failed' attributes because in RFB check it has different meaning, finally it is either pass or fail, even all testcases are set skipped, it passes but no unstable status should be reported. So I think I should check the status again in the rfb exporter.

Reply:
I think you are confusing with something else. Please can you just remove these passed/failed fields, the JSON should not contain redundant data to minimise its size.

Reply:
I just checked with Yuxuan/Yifan about the changes made for Report status. I moved the attributes.